caffeinated/payment_methods/Aim/EEG_Aim.gateway.php

```diff
@@ -446,6 +447,21 @@ private function _log_and_clean_response($response_obj, $payment)
         $this->log(array('AIM Response received:' => (array) $response_obj), $payment);
         return $response_obj;
     }
+
+    /**
+     * Removes characters Authorize.net doesn't handle well.
+     * @since $VID:$
+     * @param $text
+     * @return string
+     */
+    private function prepareStringForAuthnet($text)
+    {
+        return str_replace(
+            ['\''],
+            [''],
+            $text
+        );
```

Comment:
Not critical, but do we want to sill keep the `$search` and `$replace` as arrays with only one value in each ? Also, a bit nit-picky, but this `str_replace()` could fit in one line.

Reply:
oh ya, the arrays aren't necessary. Thanks